src/db/migrations/30-add-collector-table.ts

```diff
@@ -0,0 +1,57 @@
+import 'dotenv/config';
+import { Knex } from 'knex';
+
+import { Table } from '../db';
+import { tableNameToViewName, updateViews } from '../migration-helpers';
+
+export const up = async (knex: Knex) => {
+  await knex.schema.createTable(Table.collectors, table => {
+    table.string('id').notNullable()
+    table.primary(['id'])
+  } )
+
+  await knex.schema.createTable(Table.nftsCollectors, table => {
+    table.string('nftId') //.references('id').inTable(Table.nfts).onDelete('cascade'); // cannot enforce this because we might not have the nft yet due to order of processing
```

Comment:
Suggestions for how to get around this order dependency? It is currently problematic because we deal with transfers ahead of populating any NFTs

Reply:
i think the processor should always be inserting nfts before transfers & nftsCollectors, as in the later comment to solve this?